plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveCompressionCodecs.java

```diff
@@ -33,6 +34,18 @@ public static HiveCompressionCodec selectCompressionCodec(ConnectorSession sessi
     }
 
     public static HiveCompressionCodec selectCompressionCodec(HiveCompressionOption compressionOption, HiveStorageFormat storageFormat)
+    {
+        HiveCompressionCodec selectedCodec = preSelectCompressionCodec(compressionOption);
```

Comment:
"preselect" name is weird, since there is no second choice, just a validation.
different name?

Reply:
I named it just `selectCompressionCodec`. Is that ok?